doc/classes/ProjectSettings.xml

```diff
@@ -60,6 +60,16 @@
 				Clears the whole configuration (not recommended, may break things).
 			</description>
 		</method>
+		<method name="get_autoload_list">
+			<return type="Dictionary[]" qualifiers="const" />
```

Comment:
`return` doesn't have `qualifiers`, consider using `--doctool` instead of adding docs manually, see https://docs.godotengine.org/en/stable/contributing/documentation/updating_the_class_reference.html#updating-class-reference-when-working-on-the-engine

Reply:
Put it on the wrong thing accidentally.